en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/tls/mutual-tls-between-gateway-and-backend.md

```diff
@@ -0,0 +1,49 @@
+# Mutual TLS Between Choreo Connect and Backend
+
+In Mutual TLS, in addition to the client verifying the server, the server also verifies the client. By having Mutual TLS between Choreo Connect and the backend, both of the following security measures are applied.
+
+- The router in Choreo Connect only creates connections with a verified set of backend servers
+- Backend servers would only accept connections by a previously introduced set of router instanaces
+
+## Export the Certificates
+
+!!! Info
+    **Before you begin**
+
+    For TLS, if the backend cert has not been added to the Choreo Connect Router truststore already, use the following commands to create a key and a certificate for the backend, and add the certificate to the Router truststore.
+
+    1. Generate a private key for the backend
+
+        ```
+        openssl genpkey -out backend.key -algorithm RSA -pkeyopt rsa_keygen_bits:2048
+        ```
+
+    2. Create the certificate
+
+        The following command creates a self-signed certificate for the backend. Note the CN and SAN values. Having both in the backend cert is necessary for the router to complete hostname verification when creating a connection with the backend. 
+
+        ```
+        openssl req -x509 -new -key backend.key -out backend.pem -subj "/CN=example.com" -reqexts SAN -extensions SAN -config <(cat /etc/ssl/openssl.cnf <(printf "[SAN]\nsubjectAltName=DNS:example.com,DNS:www.example.com")) 
+        ```
+
+    3. Add the cert in `backend.pem` to the router as decribed in [Add a Certificate to Choreo Connect Router as a Trusted Certificate]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/security/tls/transport-security/#add-a-certificate-to-choreo-connect-router-as-a-trusted-certificate).
```

Comment:
Shall we point to this - https://apim.docs.wso2.com/en/latest/deploy-and-publish/deploy-on-gateway/choreo-connect/security/transport-security/#adding-certificates-to-specific-clusters?

Basically cert of the bkend can be given in 2 ways. With the project (with APIM by uploading the cert) and the way you have mentioned. Shall we recommend cert per cluster?

Reply:
fixed with https://github.com/wso2/docs-apim/pull/4771/commits/e3ec7e4064b7d6d4cb7da59269da4d9cc38897a8